crates/bindings-csharp/BSATN.Runtime/Builtins.cs

```diff
@@ -257,19 +256,49 @@ public sealed record Time(DateTimeOffset Time_) : ScheduleAt;
 
     public static implicit operator ScheduleAt(DateTimeOffset time) => new Time(time);
 
+    /// <summary>
+    /// There are 10 C# Timestamp "Ticks" per microsecond.
+    /// </summary>
+    public static readonly ulong TicksPerMicrosecond = 10;
```

Comment:
Shame that we can't use [TimeSpan.TicksPerMicrosecond](https://learn.microsoft.com/en-us/dotnet/api/system.timespan.tickspermicrosecond?view=net-9.0#system-timespan-tickspermicrosecond) in .NET Standard...

Reply:
I'm inclined to avoid polyfills for now since they're trickier to test; this is going to get rewritten in https://github.com/clockworklabs/SpacetimeDB/pull/1836 anyway. But yeah, would be nice to just have these available. Unity...